src/character.cpp

```diff
@@ -12398,13 +12399,14 @@ stat_mod Character::get_pain_penalty() const
 {
     stat_mod ret;
     int pain = get_perceived_pain();
-    if( pain <= 0 ) {
+    // if less than 10 pain, do not apply any penalties
+    if( pain <= 10 ) {
         return ret;
     }
 
     // Int and per are penalized more, 100 pain to drop stat to zero vs 140-ish for str and dex
     float penalty_mod = pain * 0.01f;
-    float lesser_penalty_mod = pain * 0.007f;
+    float lesser_penalty_mod = pain * 0.005f;
```

Comment:
since you're defining these as varialbes, maybe we should put them into global_settings.json